src/Symfony/Bundle/TwigBundle/Tests/Functional/EmptyAppTest.php

```diff
@@ -37,12 +39,13 @@ public function registerBundles()
 
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $loader->load(function ($container) {
-            $container
-                ->loadFromExtension('twig', [ // to be removed in 5.0 relying on default
-                    'strict_variables' => false,
-                ])
-            ;
+        $loader->load(static function (ContainerBuilder $container) {
+            $container->loadFromExtension('twig', [ // to be removed in 5.0 relying on default
+                'strict_variables' => false,
+                'exception_controller' => null,
+            ]);
+            $container->register('error_catcher.error_formatter', ErrorFormatter::class);
```

Comment:
ErrorFormatter does not exist anymore

Reply:
My bad, the rebase operation didn't warn me about this, fixed thanks!